base-requirements.txt

```diff
@@ -39,3 +39,5 @@ django-filter==1.1.0
 django-ordered-model==3.4.1
 django-widget-tweaks==1.4.8
 django-countries==6.1.3
+
+pygments==2.7.3  # This will be not needed when PEPs are moved to RtD.
```

Comment:
Not sure I understand this comment. Is there a plan to host the PEPs elsewhere?

Reply:
Yes, the plan is to build PEPs using Sphinx and host them on Read the Docs at peps.python.org: https://github.com/python/peps/issues/2 (June 2016).

There's a PR to do add Sphinx support, but it's a big one and hasn't been reviewed yet: https://github.com/python/peps/pull/1385 (April 2020).

Hence reviving this PR.